src/utils/dispute-utils.js

```diff
@@ -52,6 +61,20 @@ export const transformResponseDisputeAttributes = dispute => {
       }
     }),
   }
+
+  if (transformedDispute.status !== DisputesTypes.Status.Voided) {
+    return transformedDispute
+  }
+
+  // If the dispute is voided we will override certain data
+  return overrideVoidedDispute(transformedDispute)
```

Comment:
What do you think about moving all the transforms for the override into `overrideVoidedDispute`?

That way we could have something like:

```
const transformedDispute = {
  ... // normal values, as before
}

const isDisputeVoided = ...

return isDisputeVoted ? overrideVoidedDispute(transformedDispute) : transformedDispute
```

And `overrideVoidedDispute()` would handle the overrides for the status as well.